studio/pages/project/[ref]/logs-explorer/index.tsx

```diff
@@ -175,7 +175,7 @@ export const LogsExplorerPage: NextPageWithLayout = () => {
 
             const payload: UserContent = {
               name: values.name,
-              description: values.description,
+              description: values?.description ?? '',
```

Comment:
Should either consistently use `values.` or `values?.`, seems like `?` shouldn't be needed?